install/k8s/contiv/contiv_devtest.yaml

```diff
@@ -338,3 +312,62 @@ spec:
           hostPath:
             path: /opt/gopath/src/github.com/contiv/netplugin/scripts/netContain/scripts/
 ---
+# This manifest deploys the contiv-ovs pod.
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: contiv-ovs
+  namespace: kube-system
+  labels:
+    k8s-app: contiv-ovs
+spec:
+  selector:
+    matchLabels:
+      k8s-app: contiv-ovs
+  template:
+    metadata:
+      labels:
+        k8s-app: contiv-ovs
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+        # Runs ovs containers on each Kubernetes node.
+        - name: contiv-ovsdb-server
+          image: contiv/ovs:latest
+          command: ["/scripts/start-ovsdb-server.sh"]
+          securityContext:
+            privileged: false
+          volumeMounts:
+            - mountPath: /etc/openvswitch
+              name: etc-openvswitch
+              readOnly: false
+            - mountPath: /var/run
+              name: var-run
+              readOnly: false
```

Comment:
/var/run/openvswitch/db.sock . . that's only used by switch right?  thinking we don't need to declare a mount here, the switch will be on the same filesystem already right?